api/report_server.thrift

```diff
@@ -193,20 +192,17 @@ service codeCheckerDBAccess {
                                    3: optional Encoding encoding)
                                    throws (1: shared.RequestFailed requestError),
 
-  // suppress the bug
-  bool suppressBug(1: list<i64> runIds,
-                   2: i64 reportId,
-                   3: string comment)
-                   throws (1: shared.RequestFailed requestError),
-
-  // unsuppress the bug
-  bool unSuppressBug(1: list<i64> runIds,
-                     2: i64 reportId)
-                     throws (1: shared.RequestFailed requestError),
+  // change review status of a bug.
+  bool changeReviewStatus(1: i64 reportId,
+                          2: i64 status,
+                          3: string message)
+                          throws (1: shared.RequestFailed requestError),
 
-  // get suppressed bugs in a run
-  shared.SuppressBugList getSuppressedBugs(1: i64 run_id)
-                                          throws(1: shared.RequestFailed requestError),
+  // change review status of a bug.
+  bool changeReviewStatusByHash(1: string bugHash,
```

Comment:
Agreed, same as with "hiding" the bugHash behind the comment ID for the commenting feature, let's explain this better.

And the document string is not a full sentence either. :wink: